daffodil-cli/bin.NOTICE

```diff
@@ -114,31 +86,33 @@ The following NOTICE information applies to binary components distributed with t
 
 - org.scala-lang.modules.scala-parser-combinators_<VERSION>.jar
   Scala parser combinators
-  Copyright (c) 2002-2022 EPFL
-  Copyright (c) 2011-2022 Lightbend, Inc.
+  Copyright (c) 2002-2025 EPFL
+  Copyright (c) 2011-2025 Lightbend, Inc. dba Akka
 
   Scala includes software developed at
   LAMP/EPFL (https://lamp.epfl.ch/) and
-  Lightbend, Inc. (https://www.lightbend.com/).
+  Akka (https://akka.io/).
 
 - org.scala-lang.modules.scala-xml_<VERSION>.jar
   scala-xml
-  Copyright (c) 2002-2022 EPFL
-  Copyright (c) 2011-2022 Lightbend, Inc.
+  Copyright (c) 2002-2025 EPFL
+  Copyright (c) 2011-2025 Lightbend, Inc. dba Akka
 
   scala-xml includes software developed at
   LAMP/EPFL (https://lamp.epfl.ch/) and
-  Lightbend, Inc. (https://www.lightbend.com/).
+  Akka (https://akka.io).
 
-- org.scala-lang.scala-library-<VERSION>.jar
```

Comment:
I think removing scala-library-<VERSION>.jar was incorrect. When using Scala3, the old scala-library-2.13.x.jar is still pulled in as a transitive dependency to provide the bulk of the library (e.g. collections API). The scala3-library.jar provides a much smaller number of classes specific to scala 3. So even though we are using scala 3, the scala-library-2.13 jar is still used and must be listed in our notice file.